src/collections/news/2022/2022-05-21-intel-layer-5-announce-meshMark-to-quantify-cloud-native-performance/intel-layer-5-announce-meshMark-to-quantify-cloud-native-performance.md

```diff
@@ -0,0 +1,53 @@
+subtitle: ""
+date: 2022-05-21 12:15:05 +0000
+author: ITPro Today
+thumbnail: it-pro-rgb-v2-0.png
+eurl: https://www.itprotoday.com/performance-management/intel-layer-5-announce-meshmark-quantify-cloud-native-performance
+published: true
+type: News
+#product: Service Mesh Performance
+#resource: true
+---
+
+import { NewsWrapper } from "../../News.style.js";
+
+<NewsWrapper>
+
+<div className="test">
+
+The open source Service Mesh Performance project is getting a new metric called MeshMark to help organizations manage and measure cloud-native environments.
+
+In cloud-native environments, the use of service mesh technologies to connect different operations is a growing trend. What isn't always clear with a service mesh though is how well it's actually working.
+
+MeshMark, an open source effort jointly announced by Intel and service mesh startup Layer5 on May 18, looks to help organizations measure and quantify the performance and value of a service mesh deployment. MeshMark is part of the Service Mesh Performance project at the Cloud Native Computing Foundation (CNCF), which hosted its ServiceMeshCon EU event on May 18, co-located alongside KubeCon EU 2022.
+
+"We're missing some performance characteristics," Lee Calcote, founder and CEO of Layer5 and co-chair of the CNCF Technical Advisory Group (TAG) Network, said during a ServiceMeshCon EU session. "We have a need for a clear and concise way of conveying the characters and the performance of an environment."
+
+MeshMark Brings Metrics to Cloud-Native Service Mesh
+The Service Mesh Performance project at its core is an effort to define specifications for capturing the details of a cloud-native environment in a uniform and consistent way, according to Calcote. In cloud-native environments, the use of service mesh technologies to connect different operations is a growing trend. What isn't always clear with a service mesh though is how well it's actually working.
+
+MeshMark, an open source effort jointly announced by Intel and service mesh startup Layer5 on May 18, looks to help organizations measure and quantify the performance and value of a service mesh deployment. MeshMark is part of the Service Mesh Performance project at the Cloud Native Computing Foundation (CNCF), which hosted its ServiceMeshCon EU event on May 18, co-located alongside KubeCon EU 2022.
+
+"We're missing some performance characteristics," Lee Calcote, founder and CEO of Layer5 and co-chair of the CNCF Technical Advisory Group (TAG) Network, said during a ServiceMeshCon EU session. "We have a need for a clear and concise way of conveying the characters and the performance of an environment."
+
+MeshMark Brings Metrics to Cloud-Native Service Mesh. The Service Mesh Performance project at its core is an effort to define specifications for capturing the details of a cloud-native environment in a uniform and consistent way, according to Calcote.
```

Comment:
This line is repeated in line 28, please remove this.

Reply:
ahh..ohh sorry! I'll fix it asap